client/components/experiment/index.tsx

```diff
@@ -63,7 +79,7 @@ function mapStateToProps( state: AppState, ownProps?: ExperimentProps ): Experim
 	const { name: experimentName } = ownProps;
 	return {
 		isLoading: isLoading( state ),
-		variation: getVariationForUser( state, experimentName ),
+		variation: getVariationForUser( state, experimentName ) ?? undefined,
```

Comment:
Fix TS error. `getVariationForUser` can return null while `variation` prop can be undefined.